### ISSUE TITLE
Unable to create session due to validation URL failing. 

### ISSUE THREAD

**Issue by sandeepsinghal** [areed1192/interactive-broker-python-api#29]
The following error is received when I am running the "ib_client.create_session()" call .. The authentication is successful and then some APIs are working fine if I try directly on the browser. For instance : https://localhost:5000/v1/api/iserver/scanner/params

What might have happened is that Interactive Brokers have stopped supporting the "fundamentals" api which was undocumented in the first place ? 

_Would you like to make an authenticated request (Yes/No)? Yes

--------------------------------------------------------------------------------
BAD REQUEST - STATUS CODE: 404
RESPONSE URL: https://localhost:5000/v1/portal/fundamentals/landing/265598?widgets=news&lang=en
RESPONSE HEADERS: {'Date': 'Sat, 07 Aug 2021 18:16:02 GMT', 'Server': 'Apache', 'x-response-time': '0ms', 'Content-Type': 'text/html;charset=utf-8', 'Vary': 'Origin', 'Keep-Alive': 'timeout=5, max=200', 'Connection': 'Keep-Alive', 'Transfer-Encoding': 'chunked'}
RESPONSE TEXT: <html><body><h1>Resource not found</h1></body></html>
--------------------------------------------------------------------------------_

**Comment by syntaxsdev** [comment]
Not necessarily. Logging in with paper trading works just fine - just not with a regular account.
